static/app/utils/metrics/index.tsx

```diff
@@ -463,6 +463,10 @@ export function isAllowedOp(op: string) {
   return !['max_timestamp', 'min_timestamp', 'histogram'].includes(op);
 }
 
+export function isCumulativeOp(op: string = '') {
+  return ['sum', 'count', 'count_unique'].includes(op);
```

Comment:
I would explain here why this list exists.

```suggestion
  // These aggregations can't use the `min` and `max` bounds since they are based on the granularity of the graph, which differs from the one used by the graph.
  return ['sum', 'count', 'count_unique'].includes(op);
```